src/utilities/githubAPI.mjs

```diff
@@ -2,7 +2,11 @@ import { Octokit as GithubAPI } from '@octokit/rest';
 import { createActionAuth } from '@octokit/auth-action';
 /** @type import('@octokit/rest').Octokit */
 let api;
-if (process.env.CI && process.env.CI === 'true') {
+if (
+  process.env.CI &&
+  process.env.GITHUB_ACTION &&
```

Comment:
Detect `process.env.GITHUB_ACTION` as well since vercel would set `process.env.CI` too, see https://vercel.com/guides/how-do-i-resolve-a-process-env-ci-true-error

Reply:
Did you figure out which package is setting `CI` to `1`?

Reply:
See https://github.com/cypress-io/github-action/blob/9674a20f82e9e45ec75aa66038310b00e2f24657/index.js#L223